src/enrichment_facility.h

```diff
@@ -233,81 +266,126 @@ class EnrichmentFacility : public cyclus::Facility {
-                      "doc": "tails assay from the enrichment process"}
+  #pragma cyclus var { \
+    "tooltip": "tails commodity",					\
+    "doc": "tails commodity supplied by enrichment facility",		\
+    "uitype": "outcommodity" \
+  }
+  std::string tails_commod;
+  #pragma cyclus var { \
+    "default": 0.03, "tooltip": "tails assay",				\
+    "doc": "tails assay from the enrichment process" \
+  }
   double tails_assay;
-  #pragma cyclus var {"default": 1e299, "tooltip": "SWU capacity", \
-                      "doc": "separative work unit (SWU) capcity of " \
-                             "enrichment facility"}
+  #pragma cyclus var { \
+    "default": 1e299,					       \
+    "tooltip": "SWU capacity (kgSWU/month)",			       \
+    "doc": "separative work unit (SWU) capacity of enrichment " \
+           "facility (kgSWU/month) "		       \
+  }
   double swu_capacity;
-  #pragma cyclus var {"default": 1e299, "tooltip": "maximum inventory size", \
-                      "doc": "maximum inventory capacity of natural uranium in " \
-                             "the enrichment facility"}
+  #pragma cyclus var { \
+    "default": 1e299, "tooltip": "max inventory size (kg)",		\
+    "doc": "maximum total inventory of natural uranium in "		\
+           "the enrichment facility (kg)" \
+  }
   double max_inv_size;
-  #pragma cyclus var {"default": 0, "tooltip": "initial uranium reserves", \
-                      "doc": "amount of natural uranium stored at the " \
-                             "enrichment facility at the beginning of the " \
-                             "simulation"}
+ 
+  #pragma cyclus var { \
+    "default": 1.0,						\
+    "tooltip": "maximum allowed enrichment fraction",		\
+    "doc": "maximum allowed weight fraction of U235 in product",\
+    "schema": '<optional>'				     	   \
+        '          <element name="max_enrich">'			   \
+        '              <data type="double">'			   \
+        '                  <param name="minInclusive">0</param>'   \
+        '                  <param name="maxInclusive">1</param>'   \
+        '              </data>'					   \
+        '          </element>'					   \
+        '      </optional>'					   \
+  }
+  double max_enrich;
+
+  #pragma cyclus var {							\
+    "default": 0, "tooltip": "initial uranium reserves (kg)",		\
+    "doc": "amount of natural uranium stored at the enrichment "       \
+           "facility at the beginning of the simulation (kg)"		\
+  }
   double initial_reserves;
-  #pragma cyclus var {'derived_init': 'current_swu_capacity = swu_capacity;'}
+  #pragma cyclus var { \
+    "default": 1,		       \
+    "userlevel": 10,							\
+    "tooltip": "order material requests by U235 content",		\
+    "doc": "turn on preference ordering for input material "		\
+           "so that EF chooses higher U235 content first" \
+  }
+  bool order_prefs;
+  #pragma cyclus var { 'derived_init': 'current_swu_capacity = swu_capacity;' }
```

Comment:
You should remove this pragma.  `current_swu_capacity` is a intra-timestep state variable and is just used for temporary state between a bunch of function calls that occur on a single time step.  No need for it to be a managed state variable.  Also, it doesn't need a default value from the pragma because it gets set equal to swu_capacity at the beginning of every Tick anyway.